syntax = 'proto3';

package shop;

service OrdersService {
  rpc Orders(OrdersRequest) returns (OrdersResponse) {}
}

message OrdersRequest {
  string message = 1;
}

message OrdersResponse {
  string message = 1;
}